examples/binaries/calc-hash/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "demo-calc-hash"
+version = "0.1.0"
+authors = ["Gear Technologies"]
+edition = "2018"
+license = "GPL-3.0"
+workspace = "../../../"
+
+[dependencies]
+codec = { package = "parity-scale-codec", version = "3.1.2", default-features = false, features = ["derive"]}
+sha2 = { version = "0.10.2", default-features = false }
+
+[build-dependencies]
+gear-wasm-builder = { path = "../../../utils/wasm-builder" }
+
+[features]
+std = [
+  "sha2/std",
+  "codec/std",
```

Comment:
For our binaries `std` feature are not familiar one.
I don't think there are some reason to increase binary size by including other features